libs/sdk-ui-charts/src/highcharts/chart/test/ChartTransformation.test.tsx

```diff
@@ -44,7 +44,8 @@ describe("ChartTransformation", () => {
 
     function createComponent(customProps: any = {}) {
         const props = { ...defaultProps, ...customProps };
-        return <ChartTransformation {...props} />;
+        const Wrapped = withIntl(ChartTransformation);
```

Comment:
This is double wrapping the component in intl: you export the already wrapped component and wrap it here again. Is that wanted?

Reply:
I need add root for intl provider for wrapped component for test. Otherwise test failed.